src/components/shared/page-info/page-info.module.scss

```diff
@@ -1,6 +1,6 @@
 .container {
   margin-bottom: 20px;
-  margin-top: 115px;
+  margin-top: 165px;
```

Comment:
<img width="1198" alt="Screenshot 2023-11-29 at 23 51 02" src="https://github.com/grafana/k6-docs/assets/825430/b73061d7-ff32-42a3-96c7-f90d85f9b0f4">


On my screen, the margin seems too large.

Reply:
Ah good catch, thanks @ppcano! I just updated the PR, I think it's fixed now so the higher `margin-top` only applies to the JS API section.